### ISSUE TITLE
StringMap ommits field

### ISSUE THREAD

**Issue by denizzzka** [libmir/mir-ion#180]
<s>During deserialize simple JSON into `StringMap!JsonAlgebraic`:</s>

Appeal to StringMap field not returns stored value:

```D
/+ dub.sdl:
	name "issue"

    dependency "mir-ion" version="~>0.1.39"
+/

import mir.ion.conv: json2ion;
import mir.ion.deser.ion;
import mir.algebraic_alias.json: JsonAlgebraic;
import mir.string_map: StringMap;

void main()
{
    string jsonStr = `{"access_token":"secret-data","expires_in":3599,"token_type":"Bearer"}`;

    const ion = jsonStr.json2ion;

    StringMap!JsonAlgebraic token = ion.deserializeIon!(StringMap!JsonAlgebraic);

    assert(token[`access_token`] == "secret-data");
    assert(token[`expires_in`] == 3599);
    assert(token[`token_type`] == "Bearer"); // mir/string_map.d(511): No member: token_type
}
```

(I tried to add another  field to initial JSON source but it reports another unfound field)

**Comment by denizzzka** [comment]
This issue more related to `mir-algorithm`? It is need to move this report to its github repository?

**Comment by 9il** [comment]
> This issue more related to `mir-algorithm`? It is need to move this report to its github repository?

Yes, please.  If it can be reproduced without Ion by constructing token map directly.

**Comment by 9il** [comment]
```d
unittest
{
    import mir.algebraic_alias.json: JsonAlgebraic;
    import mir.string_map: StringMap;

    StringMap!JsonAlgebraic token;
    token[`access_token`] = "secret-data";
    token[`expires_in`] = 3599;
    token[`token_type`] = "Bearer";

    assert(token[`access_token`] == "secret-data");
    assert(token[`expires_in`] == 3599);
    assert(token[`token_type`] == "Bearer"); // mir/string_map.d(511): No member: token_type
}
```

**Comment by 9il** [comment]
Fixed in mir-algorithm `v3.10.75`. Thank you for the bug report!
